### PR TITLE
refactor: changed poi type names

### PR DESCRIPTION
**Pull request by lee-tammy**
I'm trying to make the POI types follow the same pattern, which is ```category?-identifier```. It was a little hard to make some of the types use this pattern. You can see all the pattern names at the bottom of the src/output.ts file. Let me know if you have any better ideas on how to standardize the poi types.

### CODE REVIEW

File: src/analysis-util.ts [hunk 1]
```diff
@@ -187,7 +188,7 @@ export function getRequiredModules(
     const pos: Position = getPosition(node);
     const arg = getArgument(node);
     if (arg === null) {
-      const poi = createPOI('Dynamic Require Arg', file, pos);
+      const poi = createPOI('dynamicArgument-require', file, pos);
```
Comment (jplaisted) [inline src/analysis-util.ts]: It might be worth making an enum somewhere.

File: src/output.ts [hunk 2]
```diff
@@ -138,25 +138,31 @@ function getColorFromType(type: string, detectionString: string): string {
...
+  'requiredModule-http2': 2,
+  'requiredModule-fs': 2,
+  'requiredModule-https': 2,
+  'requiredModule-net': 2,
+  'requiredModule-datagram': 2,
+  'requiredModule-child_process': 2,
+  'requiredModule-repl': 3,
+  'requiredModule-vm': 3,
+  'requiredModule-module': 3,
+  'error-syntax': 1,
```
Comment (kjin) [inline src/output.ts]: This seems to suggest that the type is "error" and the subtype is "syntax". Would something like `unprocessed-syntaxError` be more appropriate?

Comment (lee-tammy) [reply src/output.ts]: yeah, that's better

File: src/analysis-util.ts (hunk 1 above)
Comment (lee-tammy) [reply src/analysis-util.ts]: Decided to use constants :)

**Review by kjin — APPROVED** [review]

**Pull request merged**